src/audio.c

```diff
@@ -1304,7 +1311,7 @@ static int autx_print_pipeline(struct re_printf *pf, const struct autx *autx)
 }
 
 
-static int aurx_print_pipeline(struct re_printf *pf, const struct aurx *rx)
+static int aurx_print_pipeline(struct re_printf *pf, struct aurx *rx)
```

Comment:
we should try to avoid this

with this change, we are loosing the "const-ness".
removing const means that the function is modifying the state.

this was working with the old "struct lock", why is it not working with "mtx_t" ?